khal/khalendar/event.py

```diff
@@ -171,7 +171,7 @@ def __lt__(self, other):
         except TypeError:
             raise ValueError('Cannot compare events {} and {}'.format(start, other_start))
 
-    def update_start_end(self, start, end):
+    def update_start_end(self, start, end, allday=False):
```

Comment:
this doesn't look right, update_start_end should figure out by itself if start and end are of time date and datetime